### DISCUSSION TITLE
export / import data model ?

### DISCUSSION THREAD

**Discussion by DeanKamali** [directus/directus#9856] [Archive]
When working on a new project and trying to visualize the content structure, changes often happen to optimize workflow. Therefore I sometimes need to delete a model and later re-create it. 

Is there is a way that I could export then later import?

**Comment by rijkvanzanten** [comment]
Yep! https://docs.directus.io/reference/cli/#migrate-schema-to-a-different-environment

**Reply by DeanKamali** [reply]
Thank you
